### DISCUSSION TITLE
Scheduled CI/CD fails overnight (no changes)

### DISCUSSION THREAD

**Discussion by inwardmovement** [community/community#26892] [Actions]
<p>Scheduled CI/CD suddenly fails overnight (no changes to codebase), any idea why?<br>
</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/algosphere/algo-website/actions?query=workflow:ci-cd-main" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/9438102/181100483-94b0c05e-7c08-4d67-af52-365602959d39.png" class="thumbnail onebox-avatar" width="300" height="300">

<h3><a href="https://github.com/algosphere/algo-website/actions?query=workflow:ci-cd-main" target="_blank" rel="noopener nofollow ugc">algosphere/algo-website</a></h3>

<p>Algosphere website. Contribute to algosphere/algo-website development by creating an account on GitHub.</p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/inwardmovement">@inwardmovement</a>,</p>
<p>This should be due to the dependency is not installed, it doesn’t appear in your successful workflow run before.</p>
<blockquote>
<p>npm WARN autoprefixer@10.0.0 requires a peer of postcss@^8.0.2 but none is installed. You must install peer dependencies yourself.</p>
</blockquote>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/algosphere/algo-website/runs/1121584908?check_suite_focus=true#step:4:63" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/52772761/181100484-b6bebb2c-7f82-469c-976d-f2aa08bedca8.png" class="thumbnail onebox-avatar" width="300" height="300">

<h3><a href="https://github.com/algosphere/algo-website/runs/1121584908?check_suite_focus=true#step:4:63" target="_blank" rel="noopener nofollow ugc">algosphere/algo-website</a></h3>

<p>Algosphere website. Contribute to algosphere/algo-website development by creating an account on GitHub.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>You can specify autoprefixer version to 9.x according to this <a href="https://github.com/postcss/postcss/issues/1408" rel="noopener nofollow ugc">issue</a>, fixed as below, it works on fork repo.</p>
<pre><code class="lang-auto">    - name: Install dependencies
      run: |
        npm ci
        npm i -g gulp-cli postcss-cli autoprefixer@9 netlify-cli
</code></pre>
<p><img src="https://user-images.githubusercontent.com/52772761/181100485-db1f93e6-9f27-4782-99fc-282f8bdb3720.png" alt="image" data-base62-sha1="fCHwjZG0iVz5ZNLtKbGT2P9vwSq" width="654" height="453"></p>
<p>Thanks</p>
